src/lamplib/src/genny/toolchain.py

```diff
@@ -198,7 +198,7 @@ class ToolchainDownloader(Downloader):
     # If we were 💅 we could do the string logic here in python, but we're not that fancy.
     #
 
-    TOOLCHAIN_BUILD_ID = "77d7b8df6042f53ef392097cde191b2bfef252cc_23_05_05_02_21_38"
+    TOOLCHAIN_BUILD_ID = "patch_77d7b8df6042f53ef392097cde191b2bfef252cc_64628080562343a50196ef6f_23_05_15_18_57_15"
```

Comment:
NOTE: this is only temporary, and will be replaced by a proper build ID once the vcpkg patch with the libmongocrypt update lands.